engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -344,6 +403,137 @@ public PDDocument createPDFFromText(Reader text, int pageLimit)
             }
             return doc;
         }
+
+        public void setFont(String aFontName)
+        {
+            PDType1Font font = PagedTextToPDF.getStandardFont(aFontName);
+
+            if (font != null)
+            {
+                super.setFont(font);
+            }
+
+            this.fontName = aFontName;
+        }
+
+        private PDFont getFont(PDDocument doc, String name)
+        {
+            PDFont font = null;
+
+            if (name == null)
+            {
+                name = fontName != null ? fontName : getDefaultFont();
+            }
+
+            PDType1Font pdType1Font = PagedTextToPDF.getStandardFont(name);
+
+            if (pdType1Font != null)
+            {
+                font = pdType1Font;
+            }
+            else
+            {
+                FontMapping<TrueTypeFont> mapping = FontMappers.instance().getTrueTypeFont(name, null);
+
+                if (mapping != null && mapping.getFont() != null && !mapping.isFallback())
+                {
+                    try
+                    {
+                        font = PDType0Font.load(doc, mapping.getFont().getOriginalData());
+                    }
+                    catch (Exception e)
+                    {
+                        logger.error("Error loading mapping font: " + e.getMessage());

```

Comment:
We should proably log the `name` as well, unless it's already part of the exception message.

Also, SLF4J placeholders can be used again, instead of concatenating strings.

Reply:
Added `name` to the log message and SLF4J placeholders instead string concatenation as suggested.